independent-projects/tools/devtools-common/pom.xml

```diff
@@ -42,6 +42,12 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-devtools-codestarts</artifactId>
         </dependency>
+        <!-- Generator Mustache -->
+        <dependency>
+            <groupId>com.github.spullara.mustache.java</groupId>
```

Comment:
a specific reason you don't use qute ?

Reply:
just because I am more familiar with mustache, I have never used qute for this purpose.